Chapter05/pdbbind_nn.py

```diff
@@ -1,7 +1,9 @@
 # Use a neural network to predict the PDBBind dataset.  First load the data.
 
 import deepchem as dc
-pdbbind_tasks, pdbbind_datasets, transformers = dc.molnet.load_pdbbind(featurizer="grid", split="random", subset="core")
+featurizer = dc.feat.RdkitGridFeaturizer(voxel_width=2.0, sanitize=True, flatten=True,
```

Comment:
Will start doing some work on `RdkitGridFeaturizer` this week to get the codebase to a more sensible spot and fix our issues...